packages/task/src/node/custom/custom-task-runner-backend-module.ts

```diff
@@ -0,0 +1,37 @@
+/********************************************************************************
+ * Copyright (C) 2018 Red Hat, Inc. and others.
```

Comment:
@shyshywhy are you from `redhat`? We should update the file (if new) to your company and current year.